### PR TITLE
Denali

### PR DESCRIPTION
**Pull request by DenaliE**
Adding pics to menus, edit menus, all inputs required on create menu except pics. Changed pic migration to cascade on delete.

Delete tables in SP, migrate, then seed.

### CODE REVIEW

File: app/controllers/MenusController.php [hunk 1]
```diff
@@ -66,7 +66,7 @@ public function index()
 	 */
 	public function create()
 	{
-		return View::make('menus.createmenu');
+		return View::make('menus.createmenu')->with('menu', new Menu);
```
Comment (MoseyM) [inline app/controllers/MenusController.php]: Why are you passing an empty menu object here?

File: app/controllers/MenusController.php [hunk 2]
```diff
@@ -127,8 +140,8 @@ public function show($id)
 		return View::make('menus.menu')->with('menu', $menu);
 
 		} catch (Exception $e){
-			//replace with custom error message
-			App::abort(404);
+			//replace with custom styled error message
+			return "Sorry, menu cannot be displayed.";
```
Comment (MoseyM) [inline app/controllers/MenusController.php]: Why didn't you keep the abort method and just make a custom error message? This return isn't really user friendly

File: app/views/menus/_form.blade.php [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{{ Form::open($form_parameters) }}
```
Comment (MoseyM) [inline app/views/menus/_form.blade.php]: Im assuming this is a new file but why is it named with a starting underscore?

File: app/controllers/MenusController.php (hunk 1 above)
Comment (DenaliE) [inline app/controllers/MenusController.php]: So a the _form.blade can do double duty for the edit_menu and create menu blades.

File: app/controllers/MenusController.php (hunk 2 above)
Comment (DenaliE) [inline app/controllers/MenusController.php]: I forgot to do this. I will add it in a future pr.

File: app/views/menus/_form.blade.php (hunk 3 above)
Comment (DenaliE) [inline app/views/menus/_form.blade.php]: This is the name of an array, defined differently on edit_menu and create_menu.

**Pull request merged**